model_zoo/drug_discovery/gcn_for_property_prediction/README.md

```diff
@@ -0,0 +1,53 @@
+# Graph Convolutional Networks for Molecule Property Prediction
+
+We consider this model to be the first step stone for drug discovery with DGL. If you have used DeepChem before, you
+should find our model very similar to 
+[DeepChem's tutorial on graph convolution](https://deepchem.io/docs/notebooks/graph_convolutional_networks_for_tox21.html).
```

Comment:
Do we need to say this?

Reply:
One goal of this model is to replicate DeepChem's graph convolution on Tox21 as a GNN 101 for drug discovery. As a significant proportion of our potential audience might have used DeepChem, I think this comment might be helpful.